src/plot_api/plot_config.js

```diff
@@ -21,9 +21,9 @@ module.exports = {
     // no interactivity, for export or image generation
     staticPlot: false,
 
-    // base url for the 'Edit in Chart Studio' (aka sendDataToCloud) mode bar button
+    // base URL for the 'Edit in Chart Studio' (aka sendDataToCloud) mode bar button
     // and the showLink/sendData on-graph link
-    plotlyServerUrl: 'https://plot.ly',
+    plotlyServerURL: 'https://plot.ly',
```

Comment:
good call, that also seems more standard in the built-in API (eg `canvas.toDataURL`) though we have some other internal routines that use `...Url`, like `Drawing.setClipUrl`.